### ISSUE TITLE
Tools issues week 3, 2023

### ISSUE THREAD

**Issue by tothuhien** [usegalaxy-no/galaxyadmin#86]
There're a few tools that ended up with error. 
export_remote: the error message indicated "time out", so it seems to be not a tool's problem
kraken_taxonomy_report:  due to wrong input format from user's data
report2gest: looks like wrong user's data format, but to look more in to details since this tool has never been run successfully

basic_illumination: there was a bug but was fixed by Kjetil. However, after that, some jobs  ended up randomly with error status and message "Couldn't flush user prefs: java.util.prefs.BackingStoreException: Couldn't get file lock." even that the job seems to be done. Maybe something to be fixed with JAVA_HOME variable.

**Comment by kjetilkl** [comment]
I have already checked the JAVA_HOME variable, and it seems to be set correctly in the container. I am not sure if the error is serious, since STDOUT indicates that the tool completed successfully anyway. The error is related to persistance of Preferences, but I don't think this is necessary, and the error could perhaps be ignored in this context. However, the tool wrapper complains anyway since it uses an "[aggressive error detection] strategy (https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection)" which triggers when the word "Exception" is found in STDERR. 

I have conferred with the end user, and we have agreed to simply modify the wrapper to make it more admissive.
